cla_backend/apps/cla_auth/views.py

```diff
@@ -59,31 +60,36 @@ def dispatch(self, request, *args, **kwargs):
     def get_password_grant(self, request, data, client):
         form = ClientIdPasswordGrantForm(data, client=client)
         if not form.is_valid():
-            logger.info(
-                "login failed",
-                extra={
-                    "IP": get_ip(request),
-                    "USERNAME": request.POST.get("username"),
-                    "CLIENT_SECRET": request.POST.get("client_secret"),
-                    "HTTP_REFERER": request.META.get("HTTP_REFERER"),
-                    "HTTP_USER_AGENT": request.META.get("HTTP_USER_AGENT"),
-                },
-            )
+            extras = {
```

Comment:
As so much of this dict is the same, can we define it once and then just update it with `FORM_ERRORS` in the invalid route? Save a bit of duplication.

Reply:
Yep, sounds good....updated to reflect this now